PostNAS_ConfDialog.py

```diff
@@ -18,10 +18,12 @@
 
 from PyQt4.QtGui import QDialog
 from PyQt4.QtCore import pyqtSignature,  QSettings
+from qgis.gui import QgsAuthConfigSelect
```

Comment:
genau diese Zeile wirft in 2.8 aber einen Fehler, die nächste ist aber bereits 
`import qgis.gui`
ich teste nochmal
Prinzipiell soll das aber auch mit 2.8 laufen (dann natürlich ohne authcfg)